### DISCUSSION TITLE
Provide possible labels when ScaleType.Labels is used

### DISCUSSION THREAD

**Discussion by tlindener** [carbon-design-system/carbon-charts#1226] [Q&A]
I'm wondering if it's possible to provide the available labels when using ScaleType.Labels because if for some reason only one label is available this looks a bit weird.

![image](https://user-images.githubusercontent.com/651357/143459290-f48c6c8b-763a-4aa8-a7fd-4645c4420ee5.png)

**Comment by theiliad** [comment]
Hi @tlindener, not sure I fully understand the question. Could you pls expand on some of the details here?

E.g. use case, provided data/options, expected behaviour etc.

**Reply by tlindener** [reply]
Hi, sure will do:

I have a set of timeseries data with label data on thr y-axis.

E.g.: Labels are: "Occupied" and "Free". When the dataset only contains one of the two labels as in the screenshot above the chart (ofc) only renders one of the two labels. 

It would be nice if I could provide the possible labels so the chart could render all possible label values at the axis.

Here is a sandbox which highlights the issue:
https://codesandbox.io/s/magical-jepsen-flobw?file=/src/index.js

![image](https://user-images.githubusercontent.com/651357/148653936-5d4b0a52-d2eb-4d90-a896-a7306978ea2b.png)

**Reply by tlindener** [reply]
Thanks to @SteffenBaumann, I solved this using the domain property on an axis
